release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
 
+#### Sample Searches
 
-## Brace Yourselves, Code Freeze Is Coming
+You can use the following queries to get yourself familiarized with issues/PRs targeting the release:
 
-One of the main purposes of Code Freeze is to get issues and PRs filtered through the use of labels, with `milestone` and the `priority` label playing an important role in that.
-
-The following items help with achieving the above:
-
-* issues targeting your release should have a *milestone*, kind, and sig.
-* PRs linked to these issues should have the same labels and milestone.
-
-If the needed additions on the issues/PRs do not match the above, you can add the relevant labels.
-
-If you are not sure (mainly priority), you can comment and ask the owner or SIG leads about the status of the issue/PR (also see [How To Escalate](#how-to-escalate) on pinging options).
-
-Other regular patterns are:
-
-*Stuck PRs*: Sometimes PRs get stuck in the approval process, as every PR requires both a `/lgtm` and an `/approve` by an Owner. When you detect such cases, you should remind relevant SIG Leads / Owners to review/approve these.
-
-*CI Signal*: Checking newly-reported test failures becomes more important; you should assume that any new failure is related to the upcoming release. Bring the issue to the attention of the CI Signal lead, and assist in getting follow-up from the appropriate SIG.  Generally the CI Signal lead will track these failures, but newly opened issues may not initially have sufficient labelling to catch their attention.
-
-[The release document from the developer's guide](https://git.k8s.io/community/contributors/devel/release.md) is also a good resource on describing how developers target issues and pull requests to a milestone.
-
-### Filtering / What belongs in the milestone
+* Issues which could require early intervention
+  - [Issues in the v1.17 milestone with the `priority/critical-urgent` label](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.17+label%3Apriority%2Fcritical-urgent): `is:open is:issue milestone:v1.17 label:priority/critical-urgent`
 
-Around this point, the release is about 1 month away, and there is less and less time for PRs to make it in time.
+  - [PRs in the v1.17 milestone with the `priority/critical-urgent` label](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+milestone%3Av1.17+label%3Apriority%2Fcritical-urgent): `is:open is:pr milestone:v1.17 label:priority/critical-urgent`
 
-Filtering is one of the main functions of the Bug Triage role. It involves informing owners and SIG leads about the timelines and asking  whether an issue/PR at hand is relevant to the release, whether it should be prioritized or moved to another release via the use of `milestone`. This process involves a mix of multiple parameters such as:
```

Comment:
This part is now more in-depth explained for each stage, i.e. what is important for each stage and when we just start removing everything. If you find anything missing please let me know.